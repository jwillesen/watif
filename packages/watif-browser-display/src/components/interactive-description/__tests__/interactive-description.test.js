import React from 'react'
import {shallow} from 'enzyme'
import InteractiveDescription from '../index'

it('renders properly', () => {
  const wrapper = shallow(<InteractiveDescription
    emptyText="nothing here"
    watext={<text>description here</text>}
    verbs={[]}
    onItemClick={jest.fn()}
    onVerbClick={jest.fn()}
  />)
  expect(wrapper).toMatchSnapshot()
})
