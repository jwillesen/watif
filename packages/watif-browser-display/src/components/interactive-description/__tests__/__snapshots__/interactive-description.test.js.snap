// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  styleName="root"
>
  <div
    styleName="content"
  >
    <div
      styleName="description"
    >
      <Watext
        emptyText="nothing here"
        onItemClick={[MockFunction]}
        onVerbClick={[MockFunction]}
        verbs={Array []}
        watext={
          <text>
            description here
          </text>
        }
      />
    </div>
    <div
      styleName="verb-bar"
    >
      <VerbBar
        verbs={Array []}
      />
    </div>
  </div>
</div>
`;
